contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/spi/jobs.pxu

```diff
@@ -11,10 +11,14 @@ _summary:
     To detect if the spi device exist
 _purpose:
     Check if the SPI devices exist
+_description:
+    This test will validate the SPI devices are exposed as expected.
+    Update the SPI_DEVICES variable if you have multiple SPI devices. the default value is "/dev/spidev0.0"
+    e.g. SPI_DEVICES="/dev/spidev0.0 /dev/spidev0.1 /dev/spidev0.3"
```

Comment:
Do you think separate it by comma instead of space will be a good idea?

Reply:
fixed